docs/introduction-to-smart-contracts.rst

```diff
@@ -302,7 +302,7 @@ be rejected and not become part of the block.
 
 These blocks form a linear sequence in time and that is where the word "blockchain"
 derives from. Blocks are added to the chain in rather regular intervals - for
-Ethereum this is roughly every 17 seconds.
+Ethereum this is roughly every 12 seconds.
```

Comment:
Honestly, I would simply remove any mention of approximate block times. If you look at the etherscan chart you posted, the 17 second time was only accurate for 2016, and has not been updated since. Now, yes, after the merge, the block time seems to be a lot more stable at 12 seconds - but this may change in the future, and then we'll just have another loose end in the docs.

Reply:
I agree with @nikola-matic. Alternatively, we could also make it more vague or use a range. This is not really all that relevant to Solidity though so we may be better off not mentioning it at all.

Reply:
Thanks for the feedback. I've updated the documentation.